.github/workflows/linter.yml

```diff
@@ -0,0 +1,62 @@
+name: Linters
```

Comment:
Kindly use the right linter configuration file..... You can check this [link](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml) for the required file